content/en/contribute/medic/product-development-process/cadence-of-events.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "High-level Cadence of Events"
 linkTitle: "High-level Cadence of Events"
```

Comment:
I'd just call this "Cadence" or something like that (the text wraps in the left side menu so it's kinda messy).

Reply:
I'd suggest renaming the file to just cadence as well, but also think that `schedule` could be more appropriate.